specification/sql/resource-manager/Microsoft.Sql/preview/2018-06-01-preview/DatabaseSchema.json

```diff
@@ -0,0 +1,622 @@
+          "enum": [
+            "image",
+            "text",
+            "uniqueidentifier",
+            "date",
+            "time",
+            "datetime2",
+            "datetimeoffset",
+            "tinyint",
+            "smallint",
+            "int",
+            "smalldatetime",
+            "real",
+            "money",
+            "datetime",
+            "float",
+            "sql_variant",
+            "ntext",
+            "bit",
+            "decimal",
+            "numeric",
+            "smallmoney",
+            "bigint",
+            "hierarchyid",
+            "geometry",
+            "geography",
+            "varbinary",
+            "varchar",
+            "binary",
+            "char",
+            "timestamp",
+            "nvarchar",
+            "nchar",
+            "xml",
+            "sysname"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "ColumnDataType",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "Resource": {
+      "description": "ARM resource.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "Resource ID.",
+          "type": "string",
+          "readOnly": true
+        },
+        "name": {
+          "description": "Resource name.",
+          "type": "string",
+          "readOnly": true
+        },
+        "type": {
+          "description": "Resource type.",
+          "type": "string",
+          "readOnly": true
+        }
+      },
+      "x-ms-azure-resource": true
+    },
+    "ProxyResource": {
+      "description": "ARM proxy resource.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "properties": {}
+    },
+    "DatabaseColumn": {
+      "description": "A database column resource.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/DatabaseColumnProperties",
+          "description": "Resource properties.",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "DatabaseColumnListResult": {
+      "description": "A list of database columns.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Array of results.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DatabaseColumn"
+          },
+          "readOnly": true
+        },
+        "nextLink": {
+          "description": "Link to retrieve next page of results.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "DatabaseSchema": {
+      "description": "A database schema resource.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {}
```

Comment:
looks like schema and tables dont have any resource specific properties now after removing the child resource payload. I